src/session.rs

```diff
@@ -39,6 +39,10 @@ impl Session {
             Self::User(session) => session.id(),
         }
     }
+
+    pub fn robocorp_home(&self, robocorp_home_base: &Utf8Path) -> Utf8PathBuf {
```

Comment:
Let's remove this, since we hopefully only need to call it one place.

Reply:
It's called in three places?